ag-grid-enterprise/build.boot

```diff
@@ -0,0 +1,61 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```

Comment:
This package should depend on `cljsjs/ag-grid`, `:requires` on the `deps.cljs` alone doesn't add dependency on cljsjs package.